test/server/admin/logs_handler_test.cc

```diff
@@ -70,5 +71,90 @@ TEST_P(AdminInstanceTest, LogLevelSetting) {
   EXPECT_EQ(Http::Code::OK, postCallback("/logging?level=warning&paths=", header_map, response));
 }
 
+TEST_P(AdminInstanceTest, LogLevelFineGrainGlobSupport) {
+  Http::TestResponseHeaderMapImpl header_map;
+  Buffer::OwnedImpl response;
+
+  // now for Envoy, w/o setting the mode
+  FINE_GRAIN_LOG(info, "Build the logger for this file.");
+  Logger::Context::enableFineGrainLogger();
+  postCallback("/logging", header_map, response);
+  FINE_GRAIN_LOG(error, response.toString());
+
+  EXPECT_EQ(Http::Code::OK, postCallback("/logging?level=trace", header_map, response));
+  FINE_GRAIN_LOG(warn, "After post /logging?level=trace, all level is trace now!");
+  EXPECT_EQ(getFineGrainLogContext().getFineGrainLogEntry(__FILE__)->level(), spdlog::level::trace);
+
+  std::string query = fmt::format("/logging?{}=trace", "logs_handler_test");
```

Comment:
consider setting this to something other than trace since above makes trace the default

Reply:
Done.